src/lib.rs

```diff
@@ -404,7 +386,7 @@ impl CodeLengthReader {
                         self.result.push(0);
                     }
                 }
-                _ => abort(),
+                _ => panic!(),
```

Comment:
Wouldn't it be better to use `unreachable!()` macro? It would provide more meaningful name.